.eslintrc.js

```diff
@@ -1,5 +1,8 @@
 module.exports = {
-  'extends': ['eslint:recommended'],
+  'extends': [
+    'eslint:recommended',
+    'plugin:json/recommended'
```

Comment:
curious is this required in order to `lint` JSON files?

Reply:
yes, eslint was throwing errors without the plugin when it tries to parse the JSON